packages/api/src/useraccount/types.ts

```diff
@@ -19,7 +19,7 @@ export interface UserAccountDocument extends UserAccount, mongoose.Document {}
 
 export interface UserAccountDto {
   _id: string;
-  user?: string;
+  user?: UserDocument;
```

Comment:
Mongoose documents shoould not be a part of a Dto object meant to be transferred over http. `user` should be `string`.